0.6.0 (#43)
MetaMask/test-dapp-multichain: CHANGELOG.md

## [Unreleased]

## [0.6.0]

> ### Uncategorized

Reviewer: ### Changed

> ### Uncategorized

Reviewer: ```suggestion
### Changed
```

- Improve testability for Connected Scope related work#40 ([#41](https://github.com/MetaMask/test-dapp-multichain/pull/41))

> - small UX improvements to detected wallet UX ([#39](https://github.com/MetaMask/test-dapp-multichain/pull/39))

Reviewer 2: ```suggestion
- Improvements to detected wallet UX ([#39](https://github.com/MetaMask/test-dapp-multichain/pull/39))
```

> - for `eth_sendTransaction` calls we should just make from and to address the same by default ([#37](https://github.com/MetaMask/test-dapp-multichain/pull/37))

Reviewer 2: [nit]
```suggestion
- Make `from` and `to` addresses the same by default `eth_sendTransaction` calls ([#37](https://github.com/MetaMask/test-dapp-multichain/pull/37))
```
